subsys/bluetooth/controller/Kconfig

```diff
@@ -484,5 +484,11 @@ config BT_CTLR_SDC_SILENCE_UNEXPECTED_MSG_TYPE
 	help
 	  Convenience config to silence unexpected incoming msg type
 
+config BT_CTLR_SDC_IGNORE_HCI_ISO_DATA_TS_FROM_HOST
+	bool
+	default 0
```

Comment:
`default n` shall be omitted according to the Kconfig style guidelines: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/build/kconfig/tips.html#redundant-defaults